runelite-client/src/main/java/net/runelite/client/plugins/timetracking/TimeTrackingPlugin.java

```diff
@@ -219,6 +218,24 @@ public void checkCompletion()
 		}
 	}
 
+	@Schedule(period = 500, unit = ChronoUnit.MILLIS)
+	public void checkTimerOrder()
```

Comment:
cant you do this in updatePanel method?

Reply:
Yes - I didn't know if convention preferred one over the other